spec/integration/admin_view_spec.rb

```diff
@@ -4,14 +4,16 @@
   include Capybara::DSL
 
   before(:each) do
```

Comment:
haha, i know. i'll go in and fix it now they were my shitty tests. will just need to have emily or krista merge it tomorrow.